addons/l10n_kz/i18n/l10n_kz.pot

```diff
@@ -280,6 +280,7 @@ msgid ""
 msgstr ""
 
 #. module: l10n_kz
+#: model:account.report.expression,report_line_name:l10n_kz.l10n_kz_tr_expression_300_00_017_net
 #: model:account.report.expression,report_line_name:l10n_kz.l10n_kz_tr_expression_300_00_017_net_eeu
 #: model:account.report.expression,report_line_name:l10n_kz.l10n_kz_tr_expression_300_00_017_net_other
```

Comment:
```suggestion
```
I think we can get rid of these.

Reply:
I'm not sure if it's ok to remove these in stable? :thinking:

Reply:
I could remove these in master during fw-port for example, not sure about the taxes though, could be useful to still differentiate the transactions for some people

Reply:
I'm not 100% sure either, so let's keep them to be safe :wink: